modules/migration-installing-migration-operator-manually.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="installing-migration-operator-manually_{context}"]
+= Installing the CAM Operator manually on {product-title} 3
+
+You can install the CAM Operator manually on an {product-title} 3 source cluster, which does not support OLM.
+
+// [NOTE]
+// ====
+// You can install the CAM Operator manually on an {product-title} 4 cluster, but it is normally installed with OLM.
+// ====
+
+.Prerequisites
+
+* You must have `podman` installed.
+
+.Procedure
+
+. Download the `operator.yml` file:
++
+----
+$ podman cp $(podman create registry.redhat.io/rhcam/openshift-migration-rhel7-operator:v1.0 ):/operator.yml ./
```

Comment:
`rhcam` here should be `rhcam-1-0`, full command:

`podman cp $(podman create registry.redhat.io/rhcam-1-0/openshift-migration-rhel7-operator:v1.0 ):/operator.yml ./`